mysql-test/suite/rpl/t/rpl_semi_sync_master_disable_with_slave.cnf

```diff
@@ -0,0 +1,8 @@
+!include ../my.cnf
+
+[mysqld.1]
+rpl_semi_sync_master_enabled=1
+rpl_semi_sync_master_wait_no_slave=0
+
+[mysqld.2]
+rpl_semi_sync_slave_enabled=1
```

Comment:
Huh, does our style uses `-`s? Then why both these options only appear with `_`s in my greps?

Reply:
Either `-` or `_` works in configuration options

Reply:
Generally it is better to change the config inside the test case, as it avoids the need for mysql-test-run.pl to restart the server into the new configuration.

It is not a big issue, and if there is a reason for having a .cnf then it's fine, but I did not see in this case why it would be necessary?

Reply:
`mysql-test-run.pl` doesn’t restart the servers? Huh, I didn’t know that.